graphql/schemabuilder/reflect.go

```diff
@@ -612,8 +612,8 @@ func (sb *schemaBuilder) buildStruct(typ reflect.Type) error {
 }
 
 var scalars = map[reflect.Type]string{
-	reflect.TypeOf(bool(false)): "bool",
-	reflect.TypeOf(int(0)):      "int",
+	reflect.TypeOf(bool(false)): "Boolean",
+	reflect.TypeOf(int(0)):      "Int",
```

Comment:
The graphql spec defines an Int as an int32. Go's `int` has a platform-dependent size. So, to follow the spec to the letter, only `int32` could map to a built-in graphql scalar. However, it might be easier if we expose all int-like types as Int. (and similar for float.) What do you think?